service/src/tap_manager.rs

```diff
@@ -63,9 +64,11 @@ impl TapManager {
                 QueryError::Other(anyhow::Error::from(e))
             })?;
         if !self
-            .escrow_monitor
-            .is_sender_eligible(&receipt_signer)
+            .escrow_accounts
+            .value()
             .await
+            .map(|accounts| accounts.contains_key(&receipt_signer))
+            .unwrap_or(false)
         {
             return Err(QueryError::Other(anyhow::Error::msg(format!(
                 "Receipt's sender ({}) is not eligible for this indexer",
```

Comment:
This is too permissive. Either the balance should be checked like in https://github.com/graphops/indexer-service-rs/blob/a0b1c876ba7a60b27e8bc63925eed7deae661e1b/service/src/escrow_monitor.rs#L174-L179 or accounts with 0 balance should be filtered out early there https://github.com/graphops/indexer-service-rs/blob/72e44a258bda429f19d499a7854c1a1b2ab6342c/common/src/escrow_accounts.rs#L80-L103

Reply:
Oh! Good catch. Will fix.

Reply:
Fixed as well :)